jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -22,23 +22,39 @@
       </v-col>
     </v-row>
 
+    <!-- Composite region cannot be edited, so just grab first element. -->
+    <div v-if="is_editable">
+      <v-row v-for="(item, index2) in subset_definitions[0]" no-gutters>
+        <v-col>
+          <v-text-field
```

Comment:
Not sure whether we want to include this in scope, but I feel like this spacing feels awkwardly large to me. Thoughts @Jenneh?
![image](https://user-images.githubusercontent.com/25206008/183511300-e5412ab1-6984-450a-8455-d3566ed1c659.png)

Reply:
I'll be happy to reduce the spacing if you tell me how to do it.

I am still hesitant about the "update immediately" model. I feel like the performance hit is going to come back and bite us as we build more features. But we can discuss tomorrow.

Reply:
The extra spacing is caused by the nested `v-col`s inside `v-row`s.  When there is only a single column, its easiest to just remove `v-col` entirely.  Otherwise you'll want to add `class="row-no-outside-padding"` to the `v-row`.  See the third bullet in the [plugin style guide](https://jdaviz.readthedocs.io/en/latest/dev/ui_style_guide.html) (and feel free to clean that up if any of it seems confusing)

Reply:
I removed the `v-col` (didn't make much difference) and neither did `class="row-no-outside-padding"`, but the `class="pt-0 pb-0 mt-0 mb-0"` seems to work. I copy-pasted that from other parts of `subset_plugin.vue`.